tests/test_GreedyParameterPointPreDefinedSampler.C

```diff
@@ -0,0 +1,144 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2021, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+// Description: This source file is a test runner that uses the Google Test
+// Framework to run unit tests on the CAROM::GNAT class.
+
+#include <iostream>
+
+#ifdef CAROM_HAS_GTEST
+#include<gtest/gtest.h>
+#include <mpi.h>
+#include "../GreedyParameterPointPreDefinedSampler.h"
+#define _USE_MATH_DEFINES
+#include <cmath>
+
+/**
+ * Simple smoke test to make sure Google Test is properly linked
+ */
+TEST(GoogleTestFramework, GoogleTestFrameworkFound) {
+    SUCCEED();
+}
+
+TEST(GreedyParameterPointPreDefinedSamplerSerialTest, Test_GreedyParameterPointPreDefinedSamplerCentroid)
+{
+    // Get the rank of this process, and the number of processors.
+    int mpi_init, d_rank, d_num_procs;
+    MPI_Initialized(&mpi_init);
+    if (mpi_init == 0) {
+        MPI_Init(nullptr, nullptr);
+    }
+
+    MPI_Comm_rank(MPI_COMM_WORLD, &d_rank);
+    MPI_Comm_size(MPI_COMM_WORLD, &d_num_procs);
+
+    std::vector<double> paramPoints = {1.0, 2.0, 3.0, 99.0, 100.0, 101.0};
```

Comment:
Is there any testing of multidimensional parameters, in libROM or Laghos?

Reply:
added test here, simple for just 2 dimensions.